motioneye/mediafiles.py

```diff
@@ -1006,7 +1006,7 @@ def get_current_picture(camera_config, width, height):
     if width is height is None:
         return jpg  # no server-side resize needed
 
-    sio = io.StringIO(jpg)
+    sio = io.BytesIO(jpg) if isinstance(jpg, bytes) else io.StringIO(jpg)
```

Comment:
PIL requires a bytes object anyway, so the condition can be omitted: https://github.com/motioneye-project/motioneye/issues/2398#issuecomment-1097618544
```suggestion
    sio = io.BytesIO(jpg)
```
Needs to be changed in line 1032 below as well. The last error about truncated image in #2398 should be solved already as we switched from HTTP/1.1 to HTTP/1.0 requests to avoid chunks in the motion stream, not handled gracefully by tornado currently: #2432

I'll send a commit.